package.json

```diff
@@ -16,5 +16,8 @@
     "grunt-contrib-watch": "^1.1.0",
     "grunt-sass": "^2.1.0",
     "load-grunt-tasks": "^4.0.0"
+  },
+  "dependencies": {
+    "package-lock.json": "^1.0.0"
```

Comment:
Is this needed?

Reply:
Confirmed, this was a mistake
```bash
npm i --save package-lock.json
```

`cat node_modules/package-lock.json/package.json`
```javascript
{
  "_from": "package-lock.json",
  "_id": "package-lock.json@1.0.0",
  "_inBundle": false,
  "_integrity": "sha512-+yEXtNdlCs5N0Zy/9uvkifgf/RqnGu0WqP4j9Wu1Us4YReFe1YNBh2Krmf8B1xGxjpYnta63K55QP8bkafnOzA==",
  "_location": "/package-lock.json",
  "_phantomChildren": {},
  "_requested": {
    "type": "tag",
    "registry": true,
    "raw": "package-lock.json",
    "name": "package-lock.json",
    "escapedName": "package-lock.json",
    "rawSpec": "",
    "saveSpec": null,
    "fetchSpec": "latest"
  },
  "_requiredBy": [
    "#USER",
    "/"
  ],
  "_resolved": "https://registry.npmjs.org/package-lock.json/-/package-lock.json-1.0.0.tgz",
  "_shasum": "c4448dcd4c5a1e176b5285de4d837adb1129d437",
  "_spec": "package-lock.json",
  "_where": "/Users/gableroux/repos/npm/temp",
  "bundleDependencies": false,
  "deprecated": false,
  "description": "You probably didn’t mean to install a package.",
  "files": [],
  "name": "package-lock.json",
  "version": "1.0.0"
}
```

`cat node_modules/package-lock.json`
```
You probably didn’t mean to install a package.
```